### PR TITLE
Adding Security Features (UEFI SecureBoot and BootGuard) in NFD

### PR DESCRIPTION
**Pull request by swatisehgal**
- Discovery of security features (UEFI SecureBoot and BootGuard)
- Advertisement of node labels **node.alpha.kubernetes-incubator.io/nfd-security-UEFISecureBoot=true** and **node.alpha.kubernetes-incubator.io/nfd-security-BootGuard=true** corresponding to UEFI SecureBoot and BootGuard respectively

### CODE REVIEW

File: node-feature-discovery-job.json.template [hunk 1]
```diff
@@ -8,7 +8,8 @@
     "name": "node-feature-discovery"
   },
   "spec": {
-    "completions": COMPLETION_COUNT,
+    "completions": COMPLETION_COUNT
+,
```
Comment (okartau) [inline node-feature-discovery-job.json.template]: extra newline unintentional?

**Comment by balajismaniam** [comment]
@swatisehgal What is the status of this PR? Is it ready for a review?

**Comment by balajismaniam** [comment]
Any updates on this @swatisehgal?

**Comment by swatisehgal** [comment]
Yes, its ready for a review. It needs a rebase but I think that can done while I make updates based on reviews? Let me know if you would like me to rebase first.

**Pull request closed without merging**